src/99-appendices/11-qmri.md

```diff
@@ -0,0 +1,558 @@
+raw dataset directory along with the `MP2RAGE` file collection and to be used as the primary input
+for quantifying a `T1map`.
+
+If an additional `UNIT1` image is calculated offline, then the output is to be stored in the
+`derivatives` folder with neccesary provenance information.
+
+##### `NumberShots` metadata field
+
+Note that type of `NumberShots` field can be either a `number` or an `array of numbers`.
+
+-   If a single `number` is to be provided, this should correspond to the number of `SlicesPerSlab` or `ReconMatrixPE`.
+    However, in this case, `SlicePartialFourier` or `PartialFourierPE` fraction is needed
+    to calculate the number of partitions `before` and `after` of the k-space center to calculate a T1 map.
+
+-   If `before/after` calculation is performed during the BIDS conversion of the `MP2RAGE` data,
+    then the value of `NumberShots` metadata field can be given as a 1X2 array,
+    with first entry corresponding to `before` and the second to the `after`.
+
+Formula:
+
+If NumberShots is an array of numbers such that `"NumberShots": [before, after]`,
+the values of `before` and `after` are calculated as follows:
+
+```text
+before = SlicesPerSlab*(SlicePartialFourier - 0.5)
+after  = SlicesPerSlab/2
+```
+
+See this [reference](https://github.com/JosePMarques/MP2RAGE-related-scripts/blob/a405df30ac2c617d29d8b1b16025aaa911e86370/func/bids_T1B1correct.m#L16).
+
+##### Other metadata fields
+
+The value of the `RepetitionTimeExcitation` is not commonly found in the DICOM files.
+When accessible, the value of `EchoSpacing` corresponds to this metadata.
+When not accessible, `2 X EchoTime` can be used as a surrogate.
+
+Further information about other `MP2RAGE` qMRI protocol fields can be found [here](https://qmrlab.readthedocs.io/en/master/protocols.html#mp2rage).
+
+#### `TB1SRGE` specific notes
+
+Calculation of `before` and `after` entries for `NumberShots` metadata field of `TB1SRGE` is more involved than that of `MP2RAGE`.
+The formula can be found [here](https://github.com/JosePMarques/MP2RAGE-related-scripts/blob/a405df30ac2c617d29d8b1b16025aaa911e86370/DemoForR1Correction.m#L17),
+which requires information about `BaseResolution` (that is, image matrix size in PE direction),
+partial Fourier fraction in the PE direction, number of reference lines for parallel imaging acceleration,
+and the parallel imaging acceleration factor in PE direction.
+
+### Radiofrequency (RF) field mapping
+
+General notes:
+
+-   Some RF file collections call for the use of special notations that cannot be resolved by
+    by entities that can generalize to other applications.
+    Instead of introducing an entity that is exclusive to a single application,
+    method developers who commonly use these file collections for the `MPM` application reached
+    the consensus on the use of `acq` entity to distinguish individual files.
+    These suffixes include: `TB1AFI`, `TB1TFL`, `TB1RMF`, and `RB1COR`.
```

Comment:
This feels more like a prefatory paragraph than a bullet point.

```suggestion
Some RF file collections call for the use of special notations that cannot be resolved by
by entities that can generalize to other applications.
Instead of introducing an entity that is exclusive to a single application,
method developers who commonly use these file collections for the `MPM` application reached
the consensus on the use of `acq` entity to distinguish individual files.
These suffixes include: `TB1AFI`, `TB1TFL`, `TB1RMF`, and `RB1COR`.
```